<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillsClass</class>
 <widget class="QMainWindow" name="BillsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bills</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBoxPaid">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>See Paid</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxUnpaid">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>See Unpaid</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAll">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>See All</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Calculate Total of Unpaid Bills</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Company Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditCalculate"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonCalculate">
       <property name="text">
        <string>Calcualte</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelTotal">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidgetBills">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Bills.qrc"/>
 </resources>
 <connections/>
</ui>
